packages/gestalt/src/useResponsiveMinWidth.jsdom.test.js

```diff
@@ -0,0 +1,89 @@
+// @flow strict
+import { renderHook, act } from '@testing-library/react-hooks';
+import useResponsiveMinWidth from './useResponsiveMinWidth.js';
+
+const mediaqueryDefaults = {
+  matches: false,
+  onchange: null,
+  addEventListener: jest.fn(),
+  removeEventListener: jest.fn(),
+};
+
+describe('useReducedMotion', () => {
```

Comment:
I think the name here needs to be updated

Reply:
good catch